src/core/Decorations.ts

```diff
@@ -146,9 +191,9 @@ export default class Decorations {
     shapePenalties.default = 0;
     colorPenalties.default = 0;
 
-    const activeHatColors = HAT_COLORS.filter(
-      (color) => colorEnablement[color]
-    );
+    const activeHatColors = isTesting()
+      ? HAT_COLORS
+      : HAT_COLORS.filter((color) => colorEnablement[color]);
```

Comment:
So that unit tests don't fail locally if you have some colors disabled

Reply:
That might warrant a comment in the code?

Reply:
Marking this one unresolved until I have a chance to add the comment